content/english/csharp-guess-the-word/boolean-operators.md

```diff
@@ -26,7 +26,7 @@ To summarize, `&&` requires both Boolean expressions to be true, while `||` only
 
 ![Combining booleans](../media/booleans-advanced.png)
 
-{{% notice tip %}}
+{{% panel theme="primary" header="Instructions"%}}
```

Comment:
Question: why are we hiding the "Working Together" header within the instructions panel, but not for the "Activity" headers above? Can we be consistent?

Reply:
Good point. I'll update.